### DISCUSSION TITLE
CSV headers

### DISCUSSION THREAD

**Discussion by zizii1** [clairmont32/VirusTotal-Tools#9] [Q&A]
![Capture](https://user-images.githubusercontent.com/96738486/147494230-5b951352-59db-4251-a42e-007ded62013c.PNG)
Hello , what other columns can we add to the CSV header?
I'm using ur script for a university project.

**Comment by clairmont32** [comment]
Hi, those were the most helpful ones when I first created it. I haven't looked at their public API docs in a good long while so I'd have to check what's available without a paid license.
